multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -81,3 +81,8 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 
 # Hyperkube image repository to use.
 # hyperkubeImageRepo: quay.io/coreos/hyperkube
+
+# AWS Tags for cloudformation stack (formatted as a list of key-value pairs)
+#tags:
+#- Name: "Kubernetes" 
+#- Environment: "Production"
```

Comment:
_ahhh_ I hadn't thought of that. That simplifies [this section](https://github.com/coreos/coreos-kubernetes/pull/406/files#diff-57c0e4221c5add72bb835d26badd2241R151) significantly. Testing locally now.